docs/design/disqualification.md

```diff
@@ -0,0 +1,77 @@
+> If a storage node is disqualified, that node will no longer be selected for future data storage and the data that node stores will be moved to new storage nodes. Likewise, if a client attempts to download a piece from a storage node that the node should have stored and the node fails to return it, the node will be disqualified. Importantly, storagenodes will be allowed to reject and fail _put_ operations without penalty, as nodes will be allowed to choose which Satellite operators to work with and which data to store.
+
+'Failing to return data' is clarified to mean during an audit or a repair. Failure to return data to uplinks is specifically excluded, as this would imply a robust system of trust that does not currently exist.
+
+'Regardless of... the vetting process' is highlighted to show that both vetted and new nodes may be disqualified.
+
+> After a storage node is disqualified, the node must go back through the entire vetting process again. If the node decides to start over with a brand-new identity, the node must restart the vetting process from the beginning (in addition to generating a new nodeID via the proof of work system). This strongly disincentivizes storage nodes from being cavalier with their reputation.
+
+Further, the node will be demonetized.
+
+> Provided the storage node hasn’t been disqualified, the storage node will be paid by the Satellite for the data it has stored over the course ofthe month, per the Satellite’s records.
+
+A disqualified SNO should quickly stop particiapting with a satellite it is disqualified and demonetized on.  It may remain in Kademlia and potentially found during Overlay Cache Discover, as the kademlia network supports multiple satellites.
+
+One option that may be allowed for disqualified storage nodes is a Graceful Exit.  [Storage Node Payment and Incentives for V3](https://docs.google.com/document/d/1-Pxzk-ad-0QtF6nnTwfgzk8e_-XbBNSDxuRvnbd0QL8/edit#heading=h.rz1ehm5mbeuz) describes this feature:
+
+> When a node operator wants to leave the network, if they just shut down the node, the unavailability of the data on the network can contribute to repair costs.  If instead, the node triggers a function to call a Satellite, request new storage nodes to store the pieces stored on the node being shut down, then directly upload those pieces to the new nodes, file repair would be avoided.
+
+## Design
+
+### Use of Disqualified Nodes
+
+Disqualified nodes may be used during download of typical downloads from uplinks or via repair.  Therefore, their IP must be tracked in the overlay cache.
```

Comment:
I don't think DQ nodes should be used during regular downloads if there are enough other nodes to serve the download. I agree that the data on DQ nodes has still some value, especially if we are desperately trying to repair an irreparable segment. But other than that DQ nodes should be avoided.

Reply:
I'm picturing this as 'Graceful Exit' procedure to be the only monetized action that a DQ'ed node can take.  In the middle of the night, I realized we would need a timestamp if we did it that way, as we currently can't track these 'unpaid downloads'.  I think its fair to expect rational SNOs to never allow 'unpaid downloads' so maybe we need to initiate repair / Graceful Exit immediately.

Reply:
Please note that graceful exit was meant to not become an option for a SNO until a node has been operating for 15 months as described in blog post https://storj.io/blog/2019/01/sharing-storage-space-for-fun-and-profit/

Reply:
clarified as per architecture review